src/coreclr/jit/gentree.cpp

```diff
@@ -19978,17 +19978,21 @@ bool GenTree::SupportsSettingZeroFlag()
     }
 #endif
 #elif defined(TARGET_ARM64)
-    if (OperIs(GT_AND, GT_AND_NOT, GT_NEG))
+    if (OperIs(GT_AND, GT_AND_NOT))
     {
         return true;
     }
 
     // We do not support setting zero flag for madd/msub.
```

Comment:
```suggestion
    // We do not support setting zero flag for mneg/madd/msub.
```
feel free to skip to avoid CI rerun...